exercises/hello-world/hello_test.go

```diff
@@ -2,25 +2,17 @@ package greeting
 
 import "testing"
 
-// Define a function HelloWorld(string) string.
+// Define a function HelloWorld() string.
```

Comment:
Since a lot of later exercises do simply show the signature, I'll try to find some way to fit both in, probably, so as to transition the student a bit more smoothly.